web/sites/default/config/metatag.metatag_defaults.global.yml

```diff
@@ -15,8 +15,9 @@ tags:
   title: '[portland:title]'
   og_description: '[node:field_summary]'
   og_site_name: '[site:name]'
-  og_title: '[portland:title]'
```

Comment:
@RichardDavies, why the change from our custom token?

Reply:
The custom token wasn't working. For some reason it didn't seem to be returning any value in this context so the `og:title` element wasn't being rendered on the page. I could be wrong, but I think the custom token just appends " | Portland.gov" to the entity's title, and since the cards already include the domain name as part of the card, including this in the link title on the card is redundant anyway.